db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160607145524) do
+ActiveRecord::Schema.define(version: 20160608122002) do
```

Comment:
@lrjbrual @Feiko1  I've updated this schema with the correct id_reference keys for users (in the table restaurants) and for restaurants (in table deals) --- as we were still missing these.

**please be sure to rake db:migrate these changes**